src/words/TokTok.jl

```diff
@@ -110,19 +112,21 @@ function toktok_tokenize(instring::AbstractString)
             flush!(ts, ts.input[i:end])
         end
     end
- 
+    
     while !isdone(ts) && ts.idx <= flag
-       atoms(ts, vcat(rules_atoms...)) || 
+      atoms(ts, Tuple(Iterators.flatten(rules_atoms))) || 
```

Comment:
Do ths `Tuple(Iterators.flatten(rules_atoms)` in the definition of the constant, rather than in every iteration of the loop.